projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -197,7 +195,13 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
   public allOptions$!: Observable<QueryList<SelectOptionComponent<V>>>;
 
   public filteredOptions$!: Observable<SelectOptionComponent<V>[]>;
-  private readonly searchSubject: Subject<string> = new BehaviorSubject('');
+  private readonly caseInsensitiveSearchSubject: BehaviorSubject<string> = new BehaviorSubject('');
+
+  public isSearchTextPresent$: Observable<boolean> = this.searchValueChange.pipe(
+    map(searchText => !isEmpty(searchText)),
+    startWith(false),
+    shareReplay(1)
```

Comment:
do we need the `shareReplay` here?

Reply:
I think we do. Spent a lot of time fixing this test.